djangogirls/settings.py

```diff
@@ -274,3 +275,15 @@ def gettext(s):
 RECAPTCHA_TESTING = os.environ.get("RECAPTCHA_TESTING") == 'True'
 if RECAPTCHA_TESTING:
     SILENCED_SYSTEM_CHECKS = ['captcha.recaptcha_test_key_error']
+
+BLEACH_ALLOWED_TAGS = ['p', 'b', 'i', 'u']
+
+BLEACH_ALLOWED_ATTRIBUTES = ['title']
+
+BLEACH_ALLOWED_PROTOCOLS = [
```

Comment:
I don't think this setting is needed because links aren't allowed.

Reply:
Ok, will remove this as well.